test/to_vec.jl

```diff
@@ -67,8 +67,11 @@ function test_to_vec(x::T; check_inferred=true) where {T}
     return nothing
 end
 
+myrandn(T::Type{<:Number}, args...) = randn(T, args...)
```

Comment:
i should probably rename this to something sensible.

`robust_randn`, `randn_ints_ok`, `randn_that_doesnt_hate_ints`

Reply:
I'm not sure this should have `randn` in the name, since the integer method doesn't really sample from a normal distribution. Perhaps something a bit more verbose like `rand_number`?